modules/build/src/main/scala/scala/build/preprocessing/DataPreprocessor.scala

```diff
@@ -25,31 +25,19 @@ case object DataPreprocessor extends Preprocessor {
     input match {
       case file: VirtualData =>
         val res = either {
-          val content = new String(file.content, StandardCharsets.UTF_8)
-          val preprocessedDirectives: PreprocessedDirectives = value {
```

Comment:
This is a rather big change in behaviour although for very niche use cases.
I wonder if we should give it more thought.

Reply:
According to our documentation, we only process using directives from Scala sources - https://scala-cli.virtuslab.org/docs/guides/using-directives/. 

It seems it is bug that we process using directives from `VirtualData`.